tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -94,6 +95,17 @@ const clickFastActionFAB = async ({ actionId, waitForList }) => {
   }
 };
 
+const clickFastActionFABMobile = async ({ actionId, waitForList }) => {
```

Comment:
I am a little confused about this method. 
Did the messages test start failing?
Because if we need to add differentiation between how we click on the flat or fab button we should use a method similar to the one created by Jenny for the reports, the [openFastActionReport](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/common/common.wdio.page.js#L115-L124).

But I wonder why it stated failing 🤔

Reply:
The FAB didn't change nor was it affected by the new navigation work. The original code to handle FAB (before the Angular exception fix) works fine. These changes were reverted.

Reply:
CI is failing with `element (".fast-action-trigger .fast-action-fab-button") still not displayed after 15000ms`. 

I was facing that issue after the Angular exception fix. That's why I changed the selector to 

```
const NOT_MOBILE_ONLY = 'mm-fast-action-button:not(.mobile-only)';
const fastActionFAB = () => $(`${NOT_MOBILE_ONLY} ${FAST_ACTION_TRIGGER} .fast-action-fab-button`);
```

We had the conversation on [Slack](https://medic.slack.com/archives/D05QJT74F6E/p1725949112415229?thread_ts=1725948866.424229&cid=D05QJT74F6E)

Reply:
Yes, but it's breaking the mobile tests, so I'm doing a different approach